docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,62 @@
+# Release 403 (12 Nov 2022)
+
+## General
+
+* Add the amount of data read from external sources during table scans to
+  `EXPLAIN ANALYZE`. ({issue}`14907`)
+* Fix handling of `time(n) with time zone` when clients do not support variable
+  precision temporal types. ({issue}`14950`)
```

Comment:
Too vague what the fix is (i.e. "Fix handling" is vague) - maybe `Round `time(n) with time zone` values to precision 3 when clients do not support variable precision temporal types.`?

Reply:
The thing is, this is how it was supposed to work all along. When we introduced variable precision temporal types, clients only understood values with 3 digits of precision (and often failed with anything else depending on how they were set up to parse them). To maintain compatibility, we added a flag that presents the values to the client using at most three digits, but we missed it for time with time zone.
This change just corrects that mistake.

What we want to convey is not that the value gets rounded, but that clients can now handle (not fail) when querying time with time zone columns.

Maybe “fix potential failure when clients …”

Reply:
"Fix potential failure" makes more sense indeed.